### ISSUE TITLE
Can't compile getting SteamAuth reference error. Please help

### ISSUE THREAD

**Issue by AldarisPrime** [Jessecar96/SteamBot#1131]
The type or namespace name 'SteamAuth' could not be found (are you missing a using directive or an assembly reference?)	ExampleBot	F:\Downloads\SteamBot-master\SteamBot\Bot.cs

**Comment by AldarisPrime** [comment]
Using Visual studio 2017 to compile. I don't have much experience kindly dumb it down for me.

**Comment by Scott-GH** [comment]
Hello, if you're still struggling with this, then this may help. Visit this repository https://github.com/geel9/SteamAuth compile it and reference it in your project

**Comment by BlueRaja** [comment]
Sorry, forgot to reply here.  I was able to get it to work by removing and re-adding SteamAuth 2.0.0 form NuGet.

If someone could do that (and possibly see about upgrading to 3.0.0) and send a PR, I'd be grateful, thanks

**Comment by TimoKruth** [comment]
@BlueRaja The problem occurs not only in VS 2017 but also in Rider 17/18 sometimes. If you install something via NuGet it may not be registered correctly when building or running. Re-adding mostly does it for me in that circumstance. But couldn't yet figure out what the exact problem is.
SteamAuth 3.0.0 is no problem with the up to date version of SteamBot from my point of view. I am using it (in addition to a Rest API and some things) and it works perfectly... Should try a PR on that - when I finally get some free time - as you mentioned.

**Comment by PachiMc** [comment]
someone fixed it? same error, not working reinstalling the steamauth pack

**Comment by PachiMc** [comment]
> Sorry, forgot to reply here. I was able to get it to work by removing and re-adding SteamAuth 2.0.0 from NuGet.
> 
> Not sure why that suddenly breaks in VS 2017, but if someone could do that (and possibly see about upgrading to 3.0.0) and send a PR, I'd be grateful, thanks



> Sorry, forgot to reply here. I was able to get it to work by removing and re-adding SteamAuth 2.0.0 from NuGet.
> 
> Not sure why that suddenly breaks in VS 2017, but if someone could do that (and possibly see about upgrading to 3.0.0) and send a PR, I'd be grateful, thanks

someone fixed it? same error, not working reinstalling the steamauth pack

**Comment by JSW888** [comment]
Works for me with SteamAuth 2.0.0. You having to manually specify that version or 3.0.0 gets installed which doesn't work.

```
Install-Package SteamAuth -Version 2.0.0
```
